testing_and_setup/compass/ocean/hurricane/hurricane_wind_pressure/README

```diff
@@ -0,0 +1,22 @@
+The code in this directory is provided by Donatella Pasqualini (A-1)
```

Comment:
@sbrus89, you didn't change any of the code below Donatella wrote, did you?

Reply:
Also, please make this general and say it is provided by infrastructure team (e.g., A-1) in case others also wrote parts of the code to be more inclusive.

Reply:
@pwolfram, can you check the updated commit and see if what I have to see if if works?